src/UKHO.S100PermitService.StubService/Stubs/UserPermitsServiceStub.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Net;
+using UKHO.S100PermitService.StubService.Configuration;
+using WireMock;
+using WireMock.Matchers;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+using WireMock.Server;
+using WireMock.Types;
+using WireMock.Util;
+
+namespace UKHO.S100PermitService.StubService.Stubs
+{
+    public class UserPermitsServiceStub : IStub
+    {
+        private readonly UserPermitsServiceConfiguration _userPermitsServiceConfiguration;
+        private const string ApplicationType = "application/json";
```

Comment:
Since `ApplicationType` is used in all stubs, can we declare in separate class file which will have all common constants in one place for Stub service.

Reply:
Added new class CommonHelper.